core/server/common/src/main/java/alluxio/rocks/RocksStore.java

```diff
@@ -278,8 +288,18 @@ public synchronized void close() {
     LOG.info("Closed store at {}", mDbPath);
   }
 
-  // helper function to load RockDB configuration options based on property key configurations.
-  static Optional<BlockBasedTableConfig> checkSetTableConfig(
+  /**
+   * helper function to load RockDB configuration options based on property key configurations.
```

Comment:
Creates a helper function....
The method comment convention

Reply:
This is not going to "creates a helper function", but it **is** a helper function to do something.
Maybe we can say:
```
Loads RockDB configuration options based on property key configurations
```

Reply:
Fixed